packages/stencil/components/breadcrumb/src/components/osds-breadcrumb-item/styles/osds-breadcrumb-item.typography.scss

```diff
@@ -0,0 +1,16 @@
+@import '~@ovhcloud/ods-theming/typography/ods-theming-typography';
+@import '~@ovhcloud/ods-theming/typography/ods-component-typography';
+@import './osds-breadcrumb-item.mixins';
+
+// Main CSS mixin for typography
+@mixin osds-breadcrumb-item-theme-typography() {
+  :host([size='md']) .breadcrumb-item__wrapper {
+    //$typography-properties: ods-get-typography-properties(body, '400');
```

Comment:
do we need to keep this?

Reply:
Not really no, it's not a big deal as it doesn't generate anything and could allow to quickly update if necessary.
But I kinda hope it won't be necessary anymore after some global CSS refacto :)